res/css/views/right_panel/_TimelineCard.pcss

```diff
@@ -170,16 +170,12 @@ limitations under the License.
         flex-basis: 48px; /* 12 (padding on message list) + 36 (padding on event lines) */
     }
 
-    .mx_GenericEventListSummary_unstyledList, /* RR next to a message on the event list summary */
+    .mx_GenericEventListSummary_unstyledList,
     .mx_RoomView_MessageList {
-        /* RR next to a message on the messsge list */
         .mx_EventTile[data-layout="bubble"] {
+            /* RR next to a message on the event list summary and the messsge list */
             .mx_ReadReceiptGroup {
-                /* 6px: scroll bar width (magic number) */
-                /* stylelint-disable-next-line declaration-colon-space-after */
-                inset-inline-end: calc(
-                    -1 * var(--ReadReceiptGroup_EventBubbleTile-spacing-end) + $container-gap-width + 6px
-                );
+                inset-inline-end: var(--BaseCard_EventTile_bubble_RR-inset-inline-end);
```

Comment:
See: https://github.com/matrix-org/matrix-react-sdk/pull/10335/files#diff-5d3e25eb1d22dd7a96e0e87ea9c1298dc7dbce8eb965d30b2d1ebc1436bfec3fR1230

Reply:
> See: https://github.com/matrix-org/matrix-react-sdk/pull/10335/files#diff-5d3e25eb1d22dd7a96e0e87ea9c1298dc7dbce8eb965d30b2d1ebc1436bfec3fR1230

Once more, just linking to a diff isn't very helpful.

In this case, this is a link to the other place that `--BaseCard_EventTile_bubble_RR-inset-inline-end` is now used.